mlir/lib/Dialect/Tosa/IR/TosaOps.cpp

```diff
@@ -1778,6 +1779,15 @@ std::optional<SmallVector<int64_t, 4>> ApplyScaleOp::getShapeForUnroll() {
   return std::nullopt;
 }
 
+APFloat tosa::ReciprocalOp::computeFloatElemOne(const APFloat &floatVal,
```

Comment:
I think the name should be `computeReciprocal` or `computeReciprocalElem` as it would be clearer what it does.

Reply:
I matched the function to be same as other Tosa ops like `ReduceAllOp` to be `calcOneElement`, also added the class prefix to make it readable when calling it.